README.md

```diff
@@ -131,8 +138,28 @@ nerdctl --snapshotter nydus run -it --rm ghcr.io/stargz-containers/fedora:35-esg
 Use `crictl` to debug starting container via Kubernetes CRI. Dry run [steps](./docs/crictl_dry_run.md) of using `crictl` can be found in [documents](./docs).
 
 ### Setup with nydus-snapshotter image
+
 We can also use the `nydus-snapshotter` container image when we want to put Nydus stuffs inside a container. See the [nydus-snapshotter exmple](./misc/example/README.md) for how to setup and use it.
 
+## Integrate with Dragonfly to Distribute Images in P2P
+
+Nydus is also a sub-project of [Dragonfly](https://github.com/dragonflyoss/Dragonfly2). So it closely works with Dragonfly to distribute container images in a fast and efficient P2P fashion to reduce the pressure of Registry.
```

Comment:
`reduce network latency and lower the pressure on a single-point of the registry`?

Reply:
fixed